sql/core/src/test/scala/org/apache/spark/sql/BloomFilterAggregateQuerySuite.scala

```diff
@@ -35,23 +34,26 @@ class BloomFilterAggregateQuerySuite extends QueryTest with SharedSparkSession {
   val funcId_bloom_filter_agg = new FunctionIdentifier("bloom_filter_agg")
   val funcId_might_contain = new FunctionIdentifier("might_contain")
 
-  // Register 'bloom_filter_agg' to builtin.
-  FunctionRegistry.builtin.registerFunction(funcId_bloom_filter_agg,
```

Comment:
I think actually the root cause is that:
1. `BloomFilterAggregateQuerySuite` gets referred first somehow/somewhere
2. These functions get registered first (because these blocks will be executed when `BloomFilterAggregateQuerySuite` is referred).
3. Other tests run before `BloomFilterAggregateQuerySuite` actually runs (and deregister these functions at `afterAll`.
4. Other tests fail because of these two functions.